catkin_virtualenv/src/catkin_virtualenv/venv.py

```diff
@@ -76,9 +81,30 @@ def initialize(self, python, use_system_packages, extra_pip_args, clean=True):
         if use_system_packages:
             virtualenv.append("--system-site-packages")
 
+        without_pip = self._check_module(system_python, "ensurepip") is False
+        if without_pip:
+            virtualenv.append('--without-pip')
+
         virtualenv.append(self.path)
         run_command(virtualenv, check=True)
 
+        if without_pip:
+            # install pip via get-pip.py
+            version_proc = run_command(
+                ['python', "-cimport sys; print('{}.{}'.format(*sys.version_info))"],
```

Comment:
```suggestion
                [system_python, "-cimport sys; print('{}.{}'.format(*sys.version_info))"],
```

The merged version fails for me in a pure Python 3 environment with only a `python3` executable/symlink:

```
#19 27.48 Traceback (most recent call last):
#19 27.48   File "/opt/ros/intermodalics/lib/catkin_virtualenv/venv_init", line 46, in <module>
#19 27.48     venv.initialize(
#19 27.48   File "/opt/ros/intermodalics/lib/python3/dist-packages/catkin_virtualenv/venv.py", line 93, in initialize
#19 27.48     version_proc = run_command(
#19 27.48   File "/opt/ros/intermodalics/lib/python3/dist-packages/catkin_virtualenv/__init__.py", line 37, in run_command
#19 27.48     return subprocess.run(cmd, *args, **kwargs)
#19 27.48   File "/usr/lib/python3.8/subprocess.py", line 493, in run
#19 27.48     with Popen(*popenargs, **kwargs) as process:
#19 27.48   File "/usr/lib/python3.8/subprocess.py", line 858, in __init__
#19 27.48     self._execute_child(args, executable, preexec_fn, close_fds,
#19 27.48   File "/usr/lib/python3.8/subprocess.py", line 1704, in _execute_child
#19 27.48     raise child_exception_type(errno_num, err_msg, err_filename)
#19 27.48 FileNotFoundError: [Errno 2] No such file or directory: 'python'
```

Anyway, Python 2 support has been effectively removed in f557bbc9a262bb1d9fbdb43c3baa86f7c115ef09, but without removing the code to set variable `version`. `version_proc` and `version` became unused since then.